Plate_2_data/README.md

```diff
@@ -0,0 +1,17 @@
+# Plate 2 data 
+
+This module holds all of the analysis for data from plate 2 of the Interstellar collaboration project.
+
+## Conditions
+
+The data from plate 2 consist of modified Cell Painting images from a 384-well plate where we use 308 wells of the plate as to avoid problems with edge wells. 
+
+The Cell Painting stains include (in the correct order of the channels):
+
+- HOECHST 33342 -> Nuclei
+- PhenoVue Fluor 488 -> Concanavalin A/Endoplasmic Reticulum (ER)
+- Alexa514 -> Gasdermin (Abcam) 
+- WGA555/Phalloidin568 -> PM and Glogi/Actin
+- Mito_641_narrow -> Mitochondria
```

Comment:
```suggestion
1. HOECHST 33342 -> Nuclei
2. PhenoVue Fluor 488 -> Concanavalin A/Endoplasmic Reticulum (ER)
3. Alexa514 -> Gasdermin-D (Abcam) 
4. WGA555/Phalloidin568 -> PM and Glogi/Actin
5. Mito_641_narrow -> Mitochondria
```

You stated "in the correct order", so consider adding numbers if it makes sense. Also note `Gasdermin-D`

Reply:
Ahh that looks much better! And the Gasdermin is how it was when I got it from assay.works documentation. I will make that change to keep it consistent in the repo!

Reply:
See https://github.com/WayScience/pyroptosis_signature/pull/20/commits/fc15e3dbfbc84e2b31057b3d421d6590dda75942 for change